ntbs-service/wwwroot/css/search.scss

```diff
@@ -8,6 +8,10 @@
     margin-left: auto;
 }
 
+.search-error {
+    flex-basis: 100%;

```

Comment:
Might be worth a double check on behaviour for IE on this one

Reply:
Yep, checked this and it worked :)